### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by rfbird**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,20 @@ Once these are installed, you can configurable and build this project using
 cmake. To do so, the user should manually set the following two variables:
 
 1. `CABANA_DIR` (pointing to the install location of Cabana)
```
Comment (junghans) [inline README.md]: Do you still need this? Shouldn't https://github.com/ECP-copa/Cabana/pull/165 help to use `CMAKE_PREFIX_PATH` here as well?

File: CMakeLists.txt [hunk 2]
```diff
@@ -98,12 +101,12 @@ INCLUDE_DIRECTORIES(${CABANA_DIR}/include/Cabana/)
 LINK_DIRECTORIES(${CABANA_DIR}/lib)
 LINK_DIRECTORIES(${CABANA_DIR}/lib64)
 
-INCLUDE_DIRECTORIES(${KOKKOS_DIR}/include)
-LINK_DIRECTORIES(${KOKKOS_DIR}/lib)
+#INCLUDE_DIRECTORIES(${KOKKOS_DIR}/include)
+#LINK_DIRECTORIES(${KOKKOS_DIR}/lib)
```
Comment (junghans) [inline CMakeLists.txt]: Why not delete?

File: CMakeLists.txt [hunk 3]
```diff
@@ -98,12 +101,12 @@ INCLUDE_DIRECTORIES(${CABANA_DIR}/include/Cabana/)
...
 
 add_executable(minipic ${SOURCES} ${INPUT_DECK})
 # TODO: why is hwloc needed here for travis?
-target_link_libraries(minipic cabanacore kokkos dl hwloc)
+target_link_libraries(minipic cabanacore kokkoscore dl hwloc)
```
Comment (junghans) [inline CMakeLists.txt]: Should it be `Kokkos::kokkos`?

File: CMakeLists.txt [hunk 4]
```diff
@@ -13,6 +13,9 @@ endif()
 
 include_directories(${PROJECT_SOURCE_DIR})
 
+# find kokkos
+find_package(Kokkos 3 REQUIRED)
```
Comment (junghans) [inline CMakeLists.txt]: I think calling `find_package(Cabana)` should now incl. finding `kokkos` as well.

Comment (rfbird) [reply CMakeLists.txt]: It looks like if I just link `Cabana::cabanacore` then I get ` Target "minipic" links to target "Kokkos::kokkos" but the target was not` even though Kokkos is not on the link line

Comment (junghans) [reply CMakeLists.txt]: That would be a bug in our export target, `Cabana::cabanacore` should through `Kokkos::kokkos` in automatically.

Comment (rfbird) [reply CMakeLists.txt]: Any short term plans to fix in Cabana? I can keep my `find_pacakge(Kokkos..` for now

**Pull request merged**